packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -121,3 +161,69 @@ export function createVerifiedAttributeLogic({
 
   return toCreateEventAttributeAdded(newVerifiedAttribute);
 }
+
+export function createCertifiedAttributeLogic({
+  attribute,
+  apiCertifiedAttributeSeed,
+  certifier,
+}: {
+  attribute: WithMetadata<Attribute> | undefined;
+  apiCertifiedAttributeSeed: ApiCertifiedAttributeSeed;
+  certifier: string;
+}): CreateEvent<AttributeEvent> {
+  if (attribute) {
+    throw attributeDuplicate(apiCertifiedAttributeSeed.name);
+  }
+
+  const newCertifiedAttribute: Attribute = {
+    id: uuidv4(),
+    kind: attributeKind.certified,
+    name: apiCertifiedAttributeSeed.name,
+    description: apiCertifiedAttributeSeed.description,
+    creationTime: new Date(),
```

Comment:
is this creating the date in the same format of the as-is?

Reply:
I strongly suggest to create values directly in the desired format, otherwise we need to convert them each time they are used and it's easy to forget some cases